scripts/RunCombineFits.py

```diff
@@ -387,13 +393,14 @@ def RandomStringTag(size=6,chars=string.ascii_uppercase+string.ascii_lowercase+s
     ImpactCommand = "combineTool.py -M CollectGoodnessOfFit --input higgsCombine.saturated.GoodnessOfFit.mH125.root higgsCombine.saturated.toys.GoodnessOfFit.mH125.*.root -o "+GOFJsonName
     os.system(ImpactCommand)
 
-    ImpactCommand = "python ../../../CombineTools/scripts/plotGof.py --statistic saturated --mass 125.0 "+GOFJsonName+" --title-right='' --output='saturated' --title-left='e#mu'"
+    ImpactCommand = "python ../../../CombineTools/scripts/plotGof.py --statistic saturated --mass 125.0 "+GOFJsonName+" --title-right='' --output='saturated' --title-left='#mu#tau'"
```

Comment:
Can you change the title thing to be a bit more reflective of the channels actually in the fit?

maybe just do 5, the 4 individual channels, and then mutl-channel/all ?